frontend/benefit/handler/src/components/applicationForm/formContent/useFormContent.ts

```diff
@@ -118,6 +139,7 @@ const useFormContent = (
     clearDatesValues,
     clearAlternativeAddressValues,
     getErrorMessage,
+    displayPastApplicationDatesWarning,
```

Comment:
I guess it's a bit unconventional to output this as a function when it doesn't take any parameters and its return value is already constant within any specific render cycle.